bindings/cs/rl.net.native/rl.net.continuous_action_response.cc

```diff
@@ -25,7 +25,7 @@ API float GetContinuousActionChosenAction(reinforcement_learning::continuous_act
     return response->get_chosen_action();
 }
 
-API float GetContinuousActionChosenActionDensity(reinforcement_learning::continuous_action_response* response)
+API float GetContinuousActionChosenActionPdfValue(reinforcement_learning::continuous_action_response* response)
```

Comment:
This looks like reverting the rename you had in the previous PR, right?

Reply:
yes (see PR description):

I re-renamed the ChosenActionDensity to ChosenActionPdfValue since (afaik) both the terms pdf and density can both be used in a continuous actions context to refer to the probability density function. We want it to be clear that we are returning the value of the pdf (density) at the chosen action location